packages/preview-middleware-client/src/cpe/connector-service.ts

```diff
@@ -35,13 +35,26 @@ export class WorkspaceConnectorService {
         }
     }
 
-    private onChangeSaved(fileName: string, kind: 'delete' | 'create', changeType?: string) {
+    private onChangeSaved(fileName: string, kind: 'delete' | 'create', change: unknown = {}) {
+        const { changeType, content } = change as {
+            changeType?: string;
+            content?: {
+                templateName?: string;
+                fragmentPath?: string;
+            };
+        };
```

Comment:
Would it make sense to have less arguments for this method since fileName already exists in the change and you can take this parameter by restructuring the change as you are already doing?

Reply:
The fileName callback parameter is defined in the Storage interface 
![image](https://github.com/user-attachments/assets/df794549-6859-4ca1-adea-fa2ff87b7dd3)
This callback is called for any file, not obligatory for flex change file. Thus, fileName must be provided